Earthfile

```diff
@@ -331,7 +331,7 @@ dind-alpine:
     SAVE IMAGE --push --cache-from=earthly/dind:main earthly/dind:$DIND_ALPINE_TAG
 
 dind-ubuntu:
-    FROM ubuntu:latest
+    FROM ubuntu:20.10
```

Comment:
I think `latest` simply points to `20.10`... unless there's another release. It doesn't incrementally update between releases AFAIK

Reply:
ah it points to `20.04`, which gets updated at the same cadence; I was hoping it was somehow less frequently. It's still probably good to pin to a specific version, but it might not give us as much of a speedup as I was hoping for.